examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -182,18 +184,16 @@ def data_loader(rng: jax.random.PRNGKey, dataset: Dataset, batch_size: int, shuf
     steps_per_epoch = len(dataset) // batch_size
 
     if shuffle:
-        batch_idx = jax.random.permutation(rng, len(dataset))
+        batch_idx = np.random.permutation(len(dataset))
```

Comment:
@patil-suraj - those changes here also seem to give ca. a 2x speed-up of the overall training. Will run on the full example now to verify.

Reply:
Hmm weird on a subset of the dataset training is much faster then on the whole dataset. Training on the whole dataset doesn't really give a speed-up...

Reply:
Performance seems to be identical more or less...leaving it this way so for consistency with the MLM and T5-MLM script